www/css/app.css

```diff
@@ -98,6 +98,18 @@
     margin: 0 1rem;
 }
 
+#filesSelectionInfoAndCount p{
+    display: inline;
+}
+
+#archiveList {
+    width: 60%;
+}
```

Comment:
See codefactor warning below. You have duplicate selectors for archiveList in the same ruleset. Best to eliminate this one as the next one is more complete. (I know you have a third below in a different media type ruleset, which is fine.)